guides/release/routing/query-params.md

```diff
@@ -83,25 +83,26 @@ With this code, we have established the following behaviors:
    full router transition (i.e. it won't call `model` hooks and
    `setupController`, etc.); it will only update the URL.
 
-### link-to Helper
+### LinkTo component
 
-The `link-to` helper supports specifying query params using the
-`query-params` subexpression helper.
+The `<LinkTo />` component supports specifying query params using the `@query`
+argument, along with the `{{hash}}` helper
 
 ```handlebars
 // Explicitly set target query params
-{{#link-to "posts" (query-params direction="asc")}}Sort{{/link-to}}
+<LinkTo @route="posts" @query={{hash direction="asc"}}>Sort</LinkTo>
 
 // Binding is also supported
-{{#link-to "posts" (query-params direction=this.otherDirection)}}Sort{{/link-to}}
+<LinkTo @route="posts" @query={{hash direction=this.otherDirection}}>Sort</LinkTo>
+
 ```
 
 In the above examples, `direction` is presumably a query param property
 on the `posts` controller, but it could also refer to a `direction` property
 on any of the controllers associated with the `posts` route hierarchy,
 matching the leaf-most controller with the supplied property name.
 
-The `link-to` helper takes into account query parameters when determining
+The `<LinkTo>` component takes into account query parameters when determining
```

Comment:
`<LinkTo />`, to be consistent (I'm actually not a 100% sure if we want to consistently do `<LinkTo>` or consistently do `<LinkTo />`; but so far we have been doing the latter and I think it's better to keep it consistent and fix everything in one go later if we want to)